epochStart/bootstrap/fromLocalStorage.go

```diff
@@ -0,0 +1,198 @@
+package bootstrap
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/epochStart"
+	"github.com/ElrondNetwork/elrond-go/process/block/bootstrapStorage"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	storageFactory "github.com/ElrondNetwork/elrond-go/storage/factory"
+)
+
+func (e *epochStartBootstrap) initializeFromLocalStorage() {
+	var errNotCritical error
+	e.baseData.lastEpoch, e.baseData.shardId, e.baseData.lastRound, errNotCritical = storageFactory.FindLatestDataFromStorage(
+		e.generalConfig,
+		e.marshalizer,
+		e.workingDir,
+		e.genesisNodesConfig.ChainID,
+		e.defaultDBPath,
+		e.defaultEpochString,
+		e.defaultShardString,
+	)
+	if errNotCritical != nil {
+		log.Debug("no epoch db found in storage", "error", errNotCritical.Error())
+	} else {
+		e.baseData.storageExists = true
+		log.Debug("got last data from storage",
+			"epoch", e.baseData.lastEpoch,
+			"last round", e.baseData.lastRound,
+			"last shard ID", e.baseData.lastRound)
+	}
+}
+
+func (e *epochStartBootstrap) prepareEpochFromStorage() (Parameters, error) {
+	args := storageFactory.ArgsNewOpenStorageUnits{
+		GeneralConfig:      e.generalConfig,
+		Marshalizer:        e.marshalizer,
+		WorkingDir:         e.workingDir,
+		ChainID:            e.genesisNodesConfig.ChainID,
+		DefaultDBPath:      e.defaultDBPath,
+		DefaultEpochString: e.defaultEpochString,
+		DefaultShardString: e.defaultShardString,
+	}
+	openStorageHandler, err := storageFactory.NewStorageUnitOpenHandler(args)
+	if err != nil {
+		return Parameters{}, err
+	}
+
+	unitsToOpen := make([]string, 0)
```

Comment:
May use simple initialization instead of append.

Reply:
done.